### PR TITLE
Some refactoring on converter

### PR DESCRIPTION
**Pull request by heejaechang**
things needed

- [ ] how to handle refresh
- [ ] how to handle notebook header
- [ ] how to handle os
- [ ] how to get npm

### CODE REVIEW

**Comment by heejaechang** [comment]
@rchiodo not sure what it means by package.json needed to be updated.

**Comment by rchiodo** [comment]
Every update to the npm module requires that you increment the package.json (and package-lock.json) number by one. Otherwise npm won't publish.

**Review by rchiodo — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@vscode/jupyter-lsp-middleware",
-    "version": "0.2.29",
+    "version": "0.2.30",
```
Comment (rchiodo) [inline package.json:3]: package-lock.json has the number in it too. You should update that to 30 as well. (I usually do it by hand)

**Review by rchiodo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (heejaechang) [reply package.json:3]: okay

**Comment by heejaechang** [comment]
@rchiodo I don't have permission to check in

**Comment by rchiodo** [comment]
That's weird. I'll see if you have write permissions.

**Comment by rchiodo** [comment]
Okay I think you have write access now.

**Pull request merged**